index.html

```diff
@@ -25,10 +25,10 @@ <h1 class="header-welcome">Welcome</h1>
     </header>
```

Comment:
- [ ] As required for this project your tab must focus on each block on your project but I noticed that it's only focusing on links, icons, and buttons. Headings and other texts should be focusable too. Kindly fix this so that your project would be accessible. This is a short video showing how it's focusing on texts:  :wink: :smile: 

- [ ] If you need any suggestion from me I suggest you to use `tabindex` in your `headings` `paragraph` tags to make text , images focusable. :upside_down_face:

Reply:
Dear @Alfredbis29,
Changes have been made.